helper/flag-helpers/autopilot_flags.go

```diff
@@ -0,0 +1,103 @@
+package flaghelper
```

Comment:
What do you think about renaming the folder to match the package name: `flag-helpers`.